From a39dd82118941620c8c526175b563c305927d281 Mon Sep 17 00:00:00 2001
From: caoyi1 <caoyi1@lenovo.com>
Date: Wed, 23 Mar 2016 19:14:27 +0800
Subject: [PATCH 826/959] IKSWM-27253:input misc epl8802,moidfy attr sys file
 perimissions

Change-Id: I0579872121877bbccf4be27016406be71d4e52aa
Signed-off-by: caoyi1 <caoyi1@lenovo.com>
Reviewed-on: http://gerrit.mot.com/836485
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Robert Wiegand <Robert.P.Wiegand@motorola.com>
Reviewed-by: Daniel Moore <dtmoore@motorola.com>
Reviewed-by: Huosheng Liao <liaohs@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 drivers/input/misc/epl8802.c | 60 ++++++++++++++++++++++----------------------
 1 file changed, 30 insertions(+), 30 deletions(-)

diff --git a/drivers/input/misc/epl8802.c b/drivers/input/misc/epl8802.c
index ffeefee8bf2..74d91d7d0ec 100644
--- a/drivers/input/misc/epl8802.c
+++ b/drivers/input/misc/epl8802.c
@@ -2683,37 +2683,37 @@ static ssize_t epl_sensor_store_c_gain(struct device *dev, struct device_attribu
 
 /*----------------------------------------------------------------------------*/
 /*CTS --> S_IWUSR | S_IRUGO*/
-static DEVICE_ATTR(elan_status,					S_IROTH  | S_IWOTH, epl_sensor_show_status,  	  		NULL										);
-static DEVICE_ATTR(elan_reg,    				S_IROTH  | S_IWOTH, epl_sensor_show_reg,   				NULL										);
-static DEVICE_ATTR(mode,						S_IROTH  | S_IWOTH, NULL,   							epl_sensor_store_mode						);
-static DEVICE_ATTR(wait_time,					S_IROTH  | S_IWOTH, NULL,   					 		epl_sensor_store_wait_time					);
-static DEVICE_ATTR(set_threshold,     			S_IROTH  | S_IWOTH, NULL,   					 		epl_sensor_store_threshold					);
-static DEVICE_ATTR(cal_raw, 					S_IROTH  | S_IWOTH, epl_sensor_show_cal_raw, 	  		NULL										);
-static DEVICE_ATTR(als_enable,					S_IROTH  | S_IWOTH, NULL,   							epl_sensor_store_als_enable					);
-static DEVICE_ATTR(als_report_type,				S_IROTH  | S_IWOTH, NULL,								epl_sensor_store_als_report_type			);
-static DEVICE_ATTR(enable_sar, S_IROTH  | S_IWOTH, NULL, epl_sensor_store_ps_enable);
-static DEVICE_ATTR(ps_polling_mode,				S_IROTH  | S_IWOTH, epl_sensor_show_ps_polling,   		epl_sensor_store_ps_polling_mode			);
-static DEVICE_ATTR(als_polling_mode,			S_IROTH  | S_IWOTH, epl_sensor_show_als_polling,   		epl_sensor_store_als_polling_mode			);
-static DEVICE_ATTR(gain,						S_IROTH  | S_IWOTH, NULL,   							epl_sensor_store_gain						);
-static DEVICE_ATTR(ir_mode,						S_IROTH  | S_IWOTH, NULL,   							epl_sensor_store_ir_mode					);
-static DEVICE_ATTR(ir_drive,					S_IROTH  | S_IWOTH, NULL,   							epl_sensor_store_ir_drive					);
-static DEVICE_ATTR(ir_on,						S_IROTH  | S_IWOTH, NULL,								epl_sensor_store_ir_contrl					);
-static DEVICE_ATTR(interrupt_type,				S_IROTH  | S_IWOTH, NULL,								epl_sensor_store_interrupt_type				);
-static DEVICE_ATTR(integration,					S_IROTH  | S_IWOTH, NULL,								epl_sensor_store_integration				);
-static DEVICE_ATTR(adc,							S_IROTH  | S_IWOTH, NULL,								epl_sensor_store_adc						);
-static DEVICE_ATTR(cycle,						S_IROTH  | S_IWOTH, NULL,								epl_sensor_store_cycle						);
-static DEVICE_ATTR(ps_w_calfile,				S_IROTH  | S_IWOTH, NULL,			                    epl_sensor_store_ps_w_calfile				);
-static DEVICE_ATTR(i2c_w,                       S_IROTH  | S_IWOTH, NULL,                               epl_sensor_store_reg_write                  );
-static DEVICE_ATTR(unlock,				        S_IROTH  | S_IWOTH, NULL,			                    epl_sensor_store_unlock						);
-static DEVICE_ATTR(als_ch,				        S_IROTH  | S_IWOTH, NULL,			                    epl_sensor_store_als_ch_sel					);
-static DEVICE_ATTR(ps_cancel,				    S_IROTH  | S_IWOTH, NULL,			                    epl_sensor_store_ps_cancelation				);
-static DEVICE_ATTR(run_ps_cali, 				S_IROTH  | S_IWOTH, epl_sensor_show_ps_run_cali, 	  	NULL										);
-static DEVICE_ATTR(pdata,                       S_IROTH  | S_IWOTH, epl_sensor_show_pdata,              NULL                                        );
-static DEVICE_ATTR(als_data,                    S_IROTH  | S_IWOTH, epl_sensor_show_als_data,           NULL                                        );
-static DEVICE_ATTR(elan_renvo,                    S_IROTH  | S_IWOTH, epl_sensor_show_renvo,           NULL                                        );
-static DEVICE_ATTR(set_delay_ms, S_IROTH | S_IWOTH, epl_sensor_show_delay_ms, epl_sensor_store_delay_ms);
+static DEVICE_ATTR(elan_status, S_IRUSR | S_IRUGO, epl_sensor_show_status, NULL);
+static DEVICE_ATTR(elan_reg, S_IRUSR | S_IRUGO, epl_sensor_show_reg, NULL);
+static DEVICE_ATTR(mode, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_mode);
+static DEVICE_ATTR(wait_time, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_wait_time);
+static DEVICE_ATTR(set_threshold, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_threshold);
+static DEVICE_ATTR(cal_raw, S_IRUSR | S_IRUGO, epl_sensor_show_cal_raw, NULL);
+static DEVICE_ATTR(als_enable, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_als_enable);
+static DEVICE_ATTR(als_report_type, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_als_report_type);
+static DEVICE_ATTR(enable_sar, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_ps_enable);
+static DEVICE_ATTR(ps_polling_mode, S_IWUSR | S_IRUGO, epl_sensor_show_ps_polling, epl_sensor_store_ps_polling_mode);
+static DEVICE_ATTR(als_polling_mode, S_IWUSR | S_IRUGO, epl_sensor_show_als_polling, epl_sensor_store_als_polling_mode);
+static DEVICE_ATTR(gain, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_gain);
+static DEVICE_ATTR(ir_mode, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_ir_mode);
+static DEVICE_ATTR(ir_drive, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_ir_drive);
+static DEVICE_ATTR(ir_on, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_ir_contrl);
+static DEVICE_ATTR(interrupt_type, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_interrupt_type);
+static DEVICE_ATTR(integration, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_integration);
+static DEVICE_ATTR(adc, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_adc);
+static DEVICE_ATTR(cycle, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_cycle);
+static DEVICE_ATTR(ps_w_calfile, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_ps_w_calfile);
+static DEVICE_ATTR(i2c_w, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_reg_write);
+static DEVICE_ATTR(unlock, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_unlock);
+static DEVICE_ATTR(als_ch, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_als_ch_sel);
+static DEVICE_ATTR(ps_cancel, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_ps_cancelation);
+static DEVICE_ATTR(run_ps_cali, S_IRUSR | S_IRUGO, epl_sensor_show_ps_run_cali, NULL);
+static DEVICE_ATTR(pdata, S_IRUSR | S_IRUGO, epl_sensor_show_pdata, NULL);
+static DEVICE_ATTR(als_data, S_IRUSR | S_IRUGO, epl_sensor_show_als_data, NULL);
+static DEVICE_ATTR(elan_renvo, S_IRUSR | S_IRUGO, epl_sensor_show_renvo, NULL);
+static DEVICE_ATTR(set_delay_ms, S_IWUSR | S_IRUGO, epl_sensor_show_delay_ms, epl_sensor_store_delay_ms);
 #if ALS_DYN_INTT
-static DEVICE_ATTR(als_dyn_c_gain,              S_IROTH  | S_IWOTH, NULL,                               epl_sensor_store_c_gain);
+static DEVICE_ATTR(als_dyn_c_gain, S_IWUSR | S_IRUGO, NULL, epl_sensor_store_c_gain);
 #endif
 /*----------------------------------------------------------------------------*/
 static struct attribute *epl_sensor_attr_list[] =
-- 
2.11.0

